openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx

```diff
@@ -0,0 +1,330 @@
+            size="regular"
+            theme="primary"
+            variant="contained">
+            <FontAwesomeIcon icon="check" />
+          </Button>
+        ) : (
+          <Button
+            className={classNames('tw-w-16 tw-h-10', {
+              'tw-opacity-40': !allowAccess,
+            })}
+            data-testid="save-user"
+            size="regular"
+            theme="primary"
+            variant="contained"
+            onClick={handleSave}>
+            Create
+          </Button>
+        )}
+      </>
+    ) : null;
+  };
+
+  return (
+    <PageLayout classes="tw-max-w-full-hd tw-h-full tw-bg-white tw-py-4">
+      <h6 className="tw-heading tw-text-base">Create User</h6>
+      <Field>
+        <label className="tw-block tw-form-label tw-mb-0" htmlFor="email">
+          {requiredField('Email:')}
+        </label>
+        <input
+          className="tw-form-inputs tw-px-3 tw-py-1"
+          data-testid="email"
+          id="email"
+          name="email"
+          placeholder="email"
+          type="text"
+          value={email}
+          onChange={handleValidation}
+        />
+        {showErrorMsg.email && errorMsg('Email is required.')}
+        {showErrorMsg.validEmail && errorMsg('Email is invalid.')}
+      </Field>
+      <Field>
+        <label className="tw-block tw-form-label tw-mb-0" htmlFor="description">
+          Description:
+        </label>
+        <MarkdownWithPreview ref={markdownRef} value={description} />
+      </Field>
+      <Field>
+        <label className="tw-block tw-form-label tw-mb-0">Teams:</label>
+        <DropDown
+          className={classNames('tw-bg-white', {
+            'tw-bg-gray-100 tw-cursor-not-allowed': teams.length === 0,
+          })}
+          dropDownList={getDropdownOptions(teams) as DropDownListItem[]}
+          label="Teams"
+          selectedItems={selectedTeams as Array<string>}
+          type="checkbox"
+          onSelect={(_e, value) => selectedTeamsHandler(value)}
+        />
+      </Field>
+      <Field>
+        <label className="tw-block tw-form-label tw-mb-0" htmlFor="role">
+          Roles:
+        </label>
+        <DropDown
+          className={classNames('tw-bg-white', {
+            'tw-bg-gray-100 tw-cursor-not-allowed': roles.length === 0,
+          })}
+          dropDownList={getDropdownOptions(roles) as DropDownListItem[]}
+          label="Roles"
+          selectedItems={selectedRoles as Array<string>}
+          type="checkbox"
+          onSelect={(_e, value) => selectedRolesHandler(value)}
+        />
+      </Field>
+
+      <Field className="tw-flex tw-gap-5">
+        <div className="tw-flex tw-pt-1">
+          <label>Admin</label>
+          <div
+            className={classNames('toggle-switch', { open: isAdmin })}
+            data-testid="admin"
+            onClick={() => {
+              allowAccess && setIsAdmin((prev) => !prev);
+            }}>
```

Comment:
This should turn the `isBot` state to `false`

Reply:
Totally forgot about this 😅, great catch!